JASP-Desktop/analysis/jaspcontrolbase.h

```diff
@@ -44,7 +45,17 @@ class JASPControlBase : public QQuickItem
 		, ComponentsList
 	};
 
+	enum class Inclusive	{ MinMax, MinOnly, MaxOnly, None };
```

Comment:
Maybe it would be good (like we just discussed) to have each enum start where the last one left off?
Aka:
```
enum class Inclusive	{ None = 0, MinMax, MinOnly, MaxOnly };
enum class DropMode		{ DropNone  = static_cast<int>(Inclusive::MaxOnly) + 1, DropInsert, DropReplace };
enum class ListViewType { AssignedVariables  = static_cast<int>(DropMode::DropReplace) + 1, Interaction, AvailableVariables, RepeatedMeasures, Layers, AvailableInteraction };
enum class AssignType	{ AssignDefault  = static_cast<int>(ListViewType::AssignedVariables) + 1, AssignCross, AssignMainEffects, AssignInteraction, AssignAll2Way, AssignAll3Way, AssignAll4Way, AssignAll5Way };
```